fusor-ember-cli/app/templates/mainmenu.hbs

```diff
@@ -10,11 +10,11 @@
     <a class="navbar-brand" href="#"></a>
 
     <ul class="nav navbar-nav">
-      <li>{{#link-to 'deployments'}}EMBER-CLI DEV MODE ONLY{{/link-to}}</li>
-      <li>{{#link-to 'deployments'}}{{nameRHCI}} INSTALLER {{/link-to}}</li>
+      <li>{{#link-to 'deployments'}}FUSOR-EMBER-CLI / RHCI DEMO{{/link-to}}</li>
       <li>{{#link-to 'deployments'}}All Deployments{{/link-to}}</li>
-      <li>{{#link-to 'deployment-new.start'}}New {{nameRHCI}} Deployment{{/link-to}}</li>
+      <li>{{#link-to 'deployment-new.start'}}New Deployment{{/link-to}}</li>
+      <li>{{#link-to 'readme'}}README{{/link-to}}</li>
+      <li><a href='/tests?coverage=true'>Run Tests</a></li>
```

Comment:
note, we can run the tests from the browser. It is usually advisable to run `ember test --server` but because we are enabled `http-mocks` as the dummy data in development, this will also serve as the test dummy data.